src/ansys/dpf/core/operators/geo/elements_volume.py

```diff
@@ -11,13 +11,18 @@
 
 
 class elements_volume(Operator):
-    """Compute the volume of each element of a mesh using default shape
-    functions.
+    """Compute the measure of the Elements (volume for 3D elements, surface
+    for 2D elements or length for 1D elements) using default shape
+    functions, except for polyhedrons.
```

Comment:
@MichaelNale @rafacanton it does not say what happens in the case of polyhedrons

Reply:
It uses geometrical considerations to compute volume in that case, but I agree, it could be mentioned